### PR TITLE
Change Flash Duration to Be Based on Word Count

### PR DESCRIPTION
**Pull request by mosherc**
This changes the default duration to be based on the word count of the children. The children inner text is split by whitespace (spaces and new lines) and the number of words divided by 3 is the number of seconds that the flash will show before beginning the disappearing animation. Ex. a 30 word flash (including header, nested text inside of html elements, etc.) will show for 10s and then disappear in 0.5s.

### CODE REVIEW

File: src/molecules/Flash/Flash.tsx [hunk 1]
```diff
@@ -24,14 +24,19 @@ export const Flash: React.FC<FlashProps> = ({
...
-  const dismiss = () => {
-    setDismissed(true);
-  };
+  const ref = useRef<HTMLDivElement>(null);
+  const dismiss = () => setDismissed(true);
+
+  useEffect(() => {
+    if (!duration && ref.current && ref.current.innerText) {
+      duration = (1000 * ref.current.innerText.split(/\s+/).length) / 3;
+    }
```
Comment (mosherc) [inline src/molecules/Flash/Flash.tsx:38]: @tonytrucco-CS let me know if there should be a minimum duration. It's possible that the flash could contain only html elements (even if it's rare) in which the flash would immediately disappear.

```suggestion
    const minDefaultDuration = 3000; // or whatever
    if (!duration && ref.current && ref.current.innerText) {
      duration = Math.max((1000 * ref.current.innerText.split(/\s+/).length) / 3, minDefaultDuration);
    }
```

**Review by jennyvb143 — APPROVED** [review]
LGTM!

File: src/molecules/Flash/Flash.tsx (hunk 1 above)
Comment (tonytrucco-CS) [reply src/molecules/Flash/Flash.tsx:38]: Yeah, let's do 3 seconds as the minimum.

**Review by tonytrucco-CS — APPROVED** [review]

File: src/molecules/Flash/Flash.tsx (hunk 1 above)
Comment (mosherc) [reply src/molecules/Flash/Flash.tsx:38]: Oops GitHub didn't show your comment but it did show the approval so I went ahead and merged. I'll add this on Monday.

**Pull request merged**